### ISSUE TITLE
508-defect-4 [COGNITION]: BAM2 Debt Letters - Consider defining "TOP" in debt cards

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16787]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**VFS Point of Contact:** _Jennifer_

## User Story or Problem Statement

As a screen reader user, I would like to have the acronym TOP defined once, then referred to as "TOP" after that.

## Details

The debt cards for test user 1 show an acronym TOP. I'm not familiar with the term and wonder if it could be spelled out with "TOP" in parentheses the first time, then referred to as TOP thereafter. Screenshot attached below.

## Acceptance Criteria

- [ ] Team makes change or opts not to with an explanation

## Environment

* https://staging.va.gov/manage-va-debt/your-debt/
* Test user 1

## Screenshots or Trace Logs

![Screen Shot 2020-12-01 at 4 47 31 PM](https://user-images.githubusercontent.com/934879/100806078-73ef5180-33f5-11eb-92b0-30a7ffd89b1b.png)

**Comment by bradl3yC** [comment]
@1Copenut Ready for validation

**Comment by 1Copenut** [comment]
@bradl3yC Is there a change to this issue? I'm not seeing TOP spelled out, but I might be missing something. Please advise if I need to use another test user than 1. Thank you.

**Comment by bradl3yC** [comment]
@1Copenut Apparently I only changed this in 1 spot and not the other - Ill have a PR up in the AM

**Comment by bradl3yC** [comment]
@1Copenut this should definitely be gone now :)

**Comment by 1Copenut** [comment]
@bradl3yC Should I be looking at this issue in Staging? I'm still seeing "TOP" for test user 1's Chapter 35 education debt block from the original description.

**Comment by bradl3yC** [comment]
@1Copenut Sorry about that - I jumped the gun and the PR didn't make it in and then had an unrelated failing test in CI.  It was  today.

**Comment by 1Copenut** [comment]
@bradl3yC I took another look in Staging. Test User 1 still shows TOP on their debt history table. I've done some research and figured out that could mean "Treasury Offset Program". Maybe that's not available in the data coming back to the page? If that's the case, we can close this issue as `won't fix`.

**Comment by bradl3yC** [comment]
@1Copenut this is resolved - I think all content coming directly from the API has been replaced with approved content

**Comment by 1Copenut** [comment]
@bradl3yC Cool, thank you. I'll move this one to closed.
